src/client/components/ContactPage/ContactPage.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import style from './style.css';
+
+export const ContactPage = () => (
+    <section className={style.section}>
+        <h2 className={style.title}>Kontakt</h2>
+        <article className={style.article}>
+            Jesteśmy otwarci na wszelkiego rodzaju współpracę.<br/>
+            Zapraszamy do kontaktu mailowego i telefonicznego.<br/>
+            Odwiedź nas również na naszym profilu na facebooku.
+            <br/><br/><br/><br/>
```

Comment:
Zamiast używać 4 <br /> (dość nieprofesjonalne), owiń te 3 linie tekstu w inny element i daj mu margin-bottom

Reply:
Może być div ?

Reply:
Może być, ale chyba lepiej byłoby zamiast tego \<article> na górze dać \<section> a tu użyć \<article>

Reply:
tyle że mam już jeden section który owija wszystko. To nie problem ?

Reply:
Teoretycznie nie, ale może faktycznie skoro już jeden masz, dla tych 3 linijek nie ma sensu robić nowej sekcji. Może być div :)